server/code/setup/cpsetup

```diff
@@ -268,7 +268,7 @@ def main(argv):
     if not options.skipdbcfg:
         write_candlepin_conf(options)
     else:
-        print "** Skipping configuration file setup"
+        print("** Skipping configuration file setup")
```

Comment:
please add `from __future__ import print_function` at the very beginning of this file for best Py2 compatibility.